pkgs/development/tools/buildkit/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchFromGitHub, buildGoPackage }:
+
+buildGoPackage rec {
+  name = "buildkit${version}";
+  version = "0.3.3";
+  rev = "v${version}";
+
+  goPackagePath = "github.com/moby/buildkit";
+  subPackages = [ "cmd/buildctl" "cmd/buildkitd" ];
```

Looks like `buildkitd` [only suports Linux right now](https://github.com/moby/buildkit/blob/d1f5d3a65f1850226b58af14d59ea2e1a5e3223c/hack/dockerfiles/test.buildkit.Dockerfile#L96-L106)
```suggestion
  subPackages = [ "cmd/buildctl" ] ++ lib.optionals stdenv.isLinux [ "cmd/buildkitd" ];
```

---

Ah good point @marsam :+1: